finality-aleph/src/metrics.rs

```diff
@@ -233,11 +292,18 @@ impl<H: Key> Metrics<H> {
                 registry,
             )?,
             network_send_times,
+            validator_network_metrics,
         })));
 
         Ok(Metrics { inner })
     }
 
+    pub fn validator_network_metrics_cloned(&self) -> Option<impl NetworkCliqueMetrics> {
```

Comment:
Do you need that `Option` since you have `impl` for `Option<M>` anyway?

Reply:
And remove that `cloned` from its name. Anyone using this api don't care it it's cloned.

Reply:
Yeah, we don't need Option there